### ISSUE TITLE
Moralis typescript definitions not found

### ISSUE THREAD

**Issue by gretzke** [MoralisWeb3/issue-tracker#42]
```
$ npm install @types/moralis
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@types%2fmoralis - Not found
npm ERR! 404 
npm ERR! 404  '@types/moralis@latest' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

**Comment by tennox** [comment]
I think they just don't exist yet, the typescript definitions.
They might be included in npm package when they're ready.

**Comment by gotjoshua** [comment]
https://github.com/MoralisWeb3/issue-tracker/issues/77
cross ref

**Comment by gerardo15** [comment]
Typescript definitions are now available on moralis@0.0.32
